packages/components/src/select-control/index.tsx

```diff
@@ -26,6 +26,22 @@ function useUniqueId( idProp?: string ) {
 	return idProp || id;
 }
 
+function SelectOptions( {
```

Comment:
Extracted this out from the main component function because we need to pick props from the `options.map` and the names are going to conflict with the main component props.